project_task_parent_due_auto/tests/test_parent_due_auto.py

```
@@ -102,3 +91,11 @@ def test_creating_child_with_due_date_updates_parent(self):
             self.date_10,
             "Creating child with deadline should update parent",
         )
+
+    def test_closing_child_updates_parent(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_closing_child_updates_parent`

Add return type annotation: `None`

(ANN201)